README.md

```diff
@@ -128,7 +128,8 @@ Use `assign` to set the context
 
 ~~~elixir
 setup context do
-  Dict.put context, :necessary_key, :necessary_value
+  Dict.put(context, :necessary_key, :necessary_value)
+    |> Dict.put :many_key, ~W(white space delimited no interp)
```

Comment:
Good idea to have multiple key, values. But you can do it like this:

assign context, key1: :value1, key2: :value2